std/typecons.d

```diff
@@ -5537,9 +5537,9 @@ if (!is(T == class) && !(is(T == interface)))
 
                 auto init = typeid(T).initializer();
                 if (init.ptr is null) // null ptr means initialize to 0s
-                    memset(&source, 0, sz);
+                    (() @trusted => memset(&source, 0, sz))();
                 else
-                    memcpy(&source, init.ptr, sz);
+                    (() @trusted => memcpy(&source, init.ptr, sz))();
```

Comment:
`@trusted` for memcpy. Destination is non-null, source is valid if there has not already been memory corruption, and size does not overrun.